paper/sections/syntax.tex

```diff
@@ -502,7 +502,7 @@ \subsection{Anonymous Abstract Objects}
 
 \begin{ffcode}
 (dir "/tmp").walk
-  * ([f] (f.is-dir > @))
+  [f] (f.is-dir > @)
```

Comment:
@l3r8yJ it seems we had another place with this changes. Please, check the other part of this paragraph

Reply:
@Graur can you check again, please